src/lib/routing/router.ts

```diff
@@ -20,21 +20,47 @@ const defaultErrorHandler = (auto405) => async (err: Error, req: EReq, res: ERes
   res.withStatus(500).json({ error: err.message });
 }
 
+const preflightHandler = (autoCorsPreflight: AutoCorsPreflightOptions) => async (req: EReq, res: ERes) => {
+  if(autoCorsPreflight.trustedOrigins.length===0){
+    return;
+  }
+  let originHeaderValue :string | null = null;
+  if(autoCorsPreflight.trustedOrigins.length===1 && autoCorsPreflight.trustedOrigins[0] === '*'){
+    originHeaderValue = '*';
```

Comment:
This is actually quite a nuanced decision - if the browser says 

```
Origin: example.com
```

and the server replies with 

```
Access-Control-Allow-Origin: *
```

Then that *might* not be good enough for the browser.  It will still consider that a failure if it wants to send credentials (cookies) with the request.   Details explained in https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin.

We could consider this correct behaviour from Expressly: if you specify `[ "*" ]` as your list of trusted origins, then you really don't want to be seeing cookies from any origin on the internet - but this means there will be some preflights that won't work.  We should ensure we document this quirk in code (link to the MDN page in a comment), and also in the docs for the config option: make sure the developer knows the effect of using a * in their config.